plugins/com.google.cloud.tools.eclipse.appengine.libraries.ui/META-INF/MANIFEST.MF

```diff
@@ -1,4 +1,5 @@
 Manifest-Version: 1.0
+Bundle-ClassPath: .
```

Comment:
This addition is not necessary as the default is "."

Reply:
The question is whether we want to rely on the defaults or not. Most of our bundles do have this.

Reply:
I don't see how this redundant information helps.  This default will never change as it would result in widespread breakage.

Reply:
In that case, we should take it out of all the MANIFEST.MF where we already have this.

Reply:
And add a test to fail if it's found!